Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:erbium
+
+# make app directory
+WORKDIR /usr/src/app
+
+# use wild card to copy both of package.json and package-lock.json
+COPY package*.json ./
+
+# install app dependencies
+RUN npm install && \
+    npm install -g mocha
+
```

Comment:
```suggestion
# install solc
RUN curl -o /usr/bin/solc -fL https://github.com/ethereum/solidity/releases/download/v0.5.6/solc-static-linux \
    && chmod u+x /usr/bin/solc

```

How about adding this? If solc is installed, it will be easier when I run the tests.

Reply:
"good good" to add it for testing INT-SOL

thanks